index.bs

```diff
@@ -658,6 +652,23 @@ The [=reading change threshold|threshold=] value depends on the surrounding soft
 environment constraints, e.g., software power consumption optimizations or the underlying
 [=device sensor=]'s accuracy.
 
+## Conditions to expose sensor readings ## {#concepts-can-expose-sensor-readings}
+
+The user agent can define the list of [=conditions=] to be used to check if
```

Comment:
should we mention that these conditions are based on [=mitigation strategies=] list? Also there is a list of mandatory requirements https://w3c.github.io/sensors/#construct-sensor-object (top level browsing context, secure context) think it is also worth mentioning here.

Reply:
I would keep it as it is 😄